packages/zoe/src/zoeService/zoe.js

```diff
@@ -120,7 +116,7 @@ const makeZoeKit = (
   // The ZoeStorageManager composes and consolidates capabilities
   // needed by Zoe according to POLA.
   const {
-    zoeServiceDataAccess,
+    zoeServiceDataAccess: dataAccess,
```

Comment:
I don't know that my change here is correct. Will this restore the correct `dataAccess` on upgrade? If not, we still need to `provide` it from baggage. (But most of the tests still pass, so this change must be correct.)